### PR TITLE
clean up

### PR DESCRIPTION
**Pull request by garyunwin42**

### CODE REVIEW

File: contracts/Zdte.sol [hunk 1]
```diff
@@ -594,55 +592,52 @@ contract Zdte is Ownable, Pausable {
...
+        pure
+        returns (uint256 margin)
+    {
         margin = (
-            isPut ?
-            ((longStrike - shortStrike)/ longStrike) - longPremium + shortPremium :
-            ((shortStrike - longStrike)/ shortStrike) - longPremium + shortPremium
+            isPut
+                ? (longStrike - shortStrike) / 1e2 - longPremium + shortPremium
+                : ((shortStrike - longStrike) / shortStrike) * 1 ether / STRIKE_DECIMALS - longPremium + shortPremium
```
Comment (0xYYC) [inline contracts/Zdte.sol:603]: The correct one is formulae for call is : ( shortStrike - longStrike ) * 1e18 / shortStrike

File: contracts/Zdte.sol [hunk 2]
```diff
@@ -547,102 +471,112 @@ contract Zdte is Ownable, Pausable {
...
+        }
     }
 
     /// @notice External function to return the volatility
     /// @param strike Strike of option
-    function getVolatility(uint256 strike)
-        public
+    function getVolatility(uint256 strike) public view returns (uint256 volatility) {
+        volatility = uint256(volatilityOracle.getVolatility(oracleId, getCurrentExpiry(), strike));
+    }
```
Comment (witherblock) [inline contracts/Zdte.sol:485]: lets pass both strike and expiry to this public view for a better interface

**Review by witherblock — CHANGES_REQUESTED** [review]

File: contracts/Zdte.sol (hunk 2 above)
Comment (witherblock) [reply contracts/Zdte.sol]: Also how would u differentiate call and puts

Comment (garyunwin42) [reply contracts/Zdte.sol]: im not sure. i thought vol is gonna be independent of call / put, only a function of strike

Comment (witherblock) [reply contracts/Zdte.sol]: Ofc not, volatility is always different for calls and puts, only in ideal scenarios they remain same

**Pull request closed without merging**